org.adempiere.base/src/org/compiere/process/ProcessInfoParameter.java

```diff
@@ -282,4 +300,134 @@ public void setParameterName (String ParameterName)
+	 * @return String
+	 */
+	public String getParameterAsMultipleString() {
+		return getParameterMultipleString(getParameterAsString());
+	}
+
+	/**
+	 * Return the value of the parameter as a String with all values between commas surrounded by quotes
+	 * @return String
+	 */
+	public String getParameter_ToAsMultipleString() {
+		return getParameterMultipleString(getParameter_ToAsString());
+	}
+
+	/**
+	 * Return the value of the parameter as a String with all values between commas surrounded by quotes
+	 * @return String
+	 */
+	private String getParameterMultipleString(String param) {
+		if (Util.isEmpty(param))
+			return "";
+
+		String[] strarr = ((String) param).split(",");
+
+		StringBuilder whereValidated = new StringBuilder();
+		for (String par : strarr) {
+			if (whereValidated.length() > 0)
+				whereValidated.append(",");
+			whereValidated.append(DB.TO_STRING(par));
+		}
+
+		return whereValidated.toString();
+	}
+
+	/**
+	 * Return the value of the parameter as an array of int. Validate every value is an integer and throws NumberFormatException if one of the value is not a valid integer.
+	 * @return String
+	 */
+	public int[] getParameterAsMultipleIntArray() {
+		return getParameterAsMultipleIntArray(getParameterAsString());
+	}
+
+	/**
+	 * Return the value of the parameter To as an array of int. Validate every value is an integer and throws NumberFormatException if one of the value is not a valid integer.
+	 * @return String
+	 */
+	public int[] getParameterToAsMultipleIntArray() {
+		return getParameterAsMultipleIntArray(getParameter_ToAsString());
+	}
+
+	/**
+	 * Return the value of the parameter as an array of int. Validate every value is an integer and throws NumberFormatException if one of the value is not a valid integer.
+	 * @return String
+	 */
+	private int[] getParameterAsMultipleIntArray(String param) {
+
+		if (Util.isEmpty(param))
+			return new int[0];
+
+		return Arrays.stream(param.split(",")).mapToInt(Integer::parseInt).toArray();  
+	}
+
+	/**
+	 * Return the value of the parameter as an array of String. Validate every value is an integer and throws NumberFormatException if one of the value is not a valid integer.
+	 * @return String
+	 */
+	public String[] getParameterAsMultipleStringArray() {
+		return getParameterAsMultipleStringArray(getParameterAsString());
+	}
+
+	/**
+	 * Return the value of the parameter To as an array of String. Validate every value is an integer and throws NumberFormatException if one of the value is not a valid integer.
+	 * @return String
+	 */
+	public String[] getParameterToAsMultipleStringArray() {
+		return getParameterAsMultipleStringArray(getParameter_ToAsString());
+	}
+
+	/**
+	 * Return the value of the parameter as an array of String. Validate every value is an integer and throws NumberFormatException if one of the value is not a valid integer.
+	 * @return String
+	 */
+	private String[] getParameterAsMultipleStringArray(String param) {

```

Comment:
Hi @nmicoud , you have wrongly copy over the integer javadoc to the string version. For the array method, the method name can be simplified to getParameterAsStringArray and getParameterAsIntArray.

Reply:
hi
done